src/components/shared/Entity/Create.tsx

```diff
@@ -173,34 +158,32 @@ function EntityCreate({
                         )}
                     </Collapse>
 
-                    <form id={formID}>
-                        {!isValidating && connectorTags.length === 0 ? (
-                            <Alert severity="warning">
-                                <FormattedMessage
-                                    id={`${messagePrefix}.missingConnectors`}
-                                />
-                            </Alert>
-                        ) : connectorTags.length > 0 ? (
-                            <ErrorBoundryWrapper>
-                                <DetailsForm
-                                    connectorTags={connectorTags}
-                                    accessGrants={combinedGrants}
-                                />
-                            </ErrorBoundryWrapper>
-                        ) : null}
-
-                        {imageTag?.id ? (
-                            <ErrorBoundryWrapper>
-                                <EndpointConfig connectorImage={imageTag.id} />
-                            </ErrorBoundryWrapper>
-                        ) : null}
-
-                        {showCollections && imageTag?.id ? (
-                            <ErrorBoundryWrapper>
-                                <CollectionConfig />
-                            </ErrorBoundryWrapper>
-                        ) : null}
-                    </form>
+                    {!isValidating && connectorTags.length === 0 ? (
```

Comment:
Removing the form for now since we have so many buttons kinda hard to work out the proper UX for what happens when a user hits enter in the form.